apps/src/templates/teacherNavigation/TeacherNavigationRouter.tsx

```diff
@@ -237,7 +252,15 @@ const TeacherNavigationRouter: React.FC<TeacherNavigationRouterProps> = ({
               <ElementOrEmptyPage
                 showNoStudents={studentCount === 0}
                 showNoCurriculumAssigned={!anyStudentHasProgress}
-                element={<LessonMaterialsContainer />}
+                element={
+                  <LessonMaterialsContainer
+                    courseName={
+                      selectedSectionCourseName
+                        ? selectedSectionCourseName
+                        : 'this course'
```

Comment:
I prefer to not have a default value that we don't ever want to see. I would prefer to have `null` and handle null in `LessonMaterialsContainer`.